src/runtime_src/core/tools/common/XBHelpMenus.cpp

```diff
@@ -700,7 +700,7 @@ XBUtilities::produce_reports( xrt_core::device_collection devices,
       bool is_mfg = false;
       try {
         is_mfg = xrt_core::device_query<xrt_core::query::is_mfg>(device);
-      } catch (...) {}
+      } catch (const xrt_core::query::exception&) {}

```

Comment:
As constructed this code works for it relies on the default value.  
That said, have a blank catch body is makes it more difficult to understand what the intention is.   For this use-case (and similar ones),  it would be best to "repeat" the intention.  In this case, it would be:

```
} catch (const xrt_core::query::exception&) {
  is_mfg = false;
}
```